### DISCUSSION TITLE
Deployment namespace

### DISCUSSION THREAD

**Discussion by pohorely** [kedacore/keda#1461] [Q&A / Need Help] [answered]
Hello KEDA Community.

Can KEDA be deployed to a different namespace instead of the "keda" namespace?
Can this deployment bring some issues or is it pretty OK?

**Comment by tomkerkhove** [comment, marked as answer]
Yes, you can deploy it to any namespace that you would like!

For example, our .NET sample uses `keda-system` as a namespace - https://github.com/kedacore/sample-dotnet-worker-servicebus-queue

**Reply by zroubalik** [reply]
Yeah, just be sure that you change all the namespace references across resources

**Reply by pohorely** [reply]
Thanks for the answer! We are using helm for installation too, so it should be OK ;-)

**Reply by tomkerkhove** [reply]
Great to hear! Feel free to mark this as answered if it was helpful.
